### DISCUSSION TITLE
Linkedin ODIC auth in Supabase

### DISCUSSION THREAD

**Discussion by Nanobot234** [supabase/supabase#22926] [Questions]
hello Everyone. So I have been trying to implement Linkedin auth in my ios app but I don't think there is integration with the new Open ID Connect. Has this been created yet? If so how can I use it? Thanks!

**Comment by GaryAustin1** [comment]
https://supabase.com/docs/guides/auth/social-login/auth-linkedin

Have you gone thru that?

**Comment by Nanobot234** [comment]
Hello Gary,

yes I did. So that docueention doesn't have any Swift implementation as far
as I can see. And when I use the oauth API for swift it only shows the old
linkeidn As the provider.

**Reply by GaryAustin1** [reply]
I don't the swift client, and I don't think you mentioned it.

I believe the clients should not care (once again not knowing the swift client) as it is just a provider string to pass on.
![image](https://github.com/supabase/supabase/assets/54564956/1a414005-da67-4aab-aa09-c9e5281673a3)
And sorry if I have missed your issue, as I don't use OAuth myself.

**Comment by Nanobot234** [comment]
ok i understand!

**Reply by Ja4Dev** [reply]
Hey there, Nano! Have you found any solution for this problem about linkedin_oidc implementation in Swift? I'm having the same problem

**Comment by Nanobot234** [comment]
Hello ! No I haven’t found it yet. Have u asked anyone elseSent from my iPhoneOn Jun 6, 2024, at 9:00 AM, Javier Leone ***@***.***> wrote:﻿
Hey there, Nano! Have you found any solution for this problem about linkedin_oidc implementation in Swift? I'm having the same problem
